boards/arm/nrf52840_mdk/Kconfig.defconfig

```diff
@@ -22,6 +29,19 @@ config PWM_0
 
 endif # PWM
 
+if SPI
+
+config SPI_1
+	default y
```

Comment:
These SPI instances should not be enabled by default without providing proper configuration of corresponding `spiX` nodes in dts. Otherwise, you'll get compilation errors (undeclared `DT_*` symbols) for them after enabling `SPI` in Kconfig.

Reply:
My bad. Got it.